### ISSUE TITLE
Formatted messages are displayed literally

### ISSUE THREAD

**Issue by redsky17** [Nheko-Reborn/nheko#2]
Copied from https://github.com/mujx/nheko/issues/457

**Actual behavior**
Formatted message is displayed literally.

**Expected behavior**
Formatting applied with the markup hidden.

**Steps to reproduce**
Send message with the following content:

Test list:
1. One
2. Two

What is displayed is this:

```
Test list:
<ol>
<li>One</li>
<li>Two</li>
></ol>
```

**Comment by redsky17** [comment]
After some testing and perusing of the code, it appears that this might only impact outgoing messages.  Incoming messages are correctly formatted.  This makes me think the likely culprit is the TimeLineItem function used for local messages, which starts on line 230 of TimeLineItem.cpp

**Comment by redsky17** [comment]
After even more investigation, it appears that the HTML between outgoing lists and incoming lists is slightly different.  For example, here's the HTML associated with remote (incoming) event with the above text:

```
<p>Test</p>
<ol>
<li>One</li>
<li>Two</li>
</ol>
```

Here's the HTML when sending the same message locally (outgoing):

```
<p>Test</p>
<ol>
<li>One</li>
<li>Two</li>
</ol>
```

**Comment by redsky17** [comment]
It appears that this issue is due to intentionally trimming <p> tags in the markdownToHtml function.  This was added here:
https://github.com/Nheko-Reborn/nheko/commit/bf4d559523e956523a617213fa1cc3c36fab24e4
as a fix for mujx#438.  I will need to do some additional investigating to figure out how to fix this and also keep mujx#438 closed.
